app/models/notification.rb

```diff
@@ -2,8 +2,31 @@ class Notification < ActiveRecord::Base
   belongs_to :event
   # has_many :recipients
 
-  VALID_STATUSES = %w(pending active canceled complete expired)
-  validates :status, inclusion: { in: VALID_STATUSES }
```

Comment:
Since I moved it to `enum`, there's no need for a separate validation, just to ensure `status` stays within valid values.